libs/product/src/models/product.ts

```diff
@@ -16,14 +16,23 @@ export enum DaffProductTypeEnum {
  */
 export interface DaffProduct extends DaffLocatable {
 	id: ID;
-	type?: DaffProductTypeEnum;
+	type: DaffProductTypeEnum;
```

Comment:
Can you move the unrelated model changes to another PR?

Reply:
removed